validphys2/src/validphys/core.py

```diff
@@ -586,10 +594,6 @@ def to_unweighted(self):
         )
         return self
 
-    @functools.lru_cache
-    def load_commondata(self):
-        return self.commondata.load()
-
```

Comment:
Why has this been removed? Is it not used anywhere anymore?

Reply:
Yes exactly, this shouldn't be used anywhere else.